src/test/java/org/folio/edge/patron/utils/PatronOkapiClientTest.java

```diff
@@ -213,7 +215,32 @@ public void testGetAccountOffsetIsWrong(TestContext context) throws Exception {
         resp -> {
           logger.info("mod-patron response body: " + resp.body());
           context.assertEquals(400, resp.statusCode());
-          context.assertEquals(wrongOffsetMessage, resp.bodyAsString());
+          context.assertEquals(String.format(WRONG_INTEGER_PARAM_MESSAGE, OFFSET, "-1"), resp.bodyAsString());
+          async.complete();
+        },
+        context::fail);
+    });
+  }
+
+  @Test
+  public void testGetAccountLimitIsWrong(TestContext context) throws Exception {
```

Comment:
This test seems to be checking the specific case that the limit is negative. I think the name could refer to that rather than the general idea of the parameter being wrong.

```suggestion
  public void testGetAccountLimitIsNegative(TestContext context) throws Exception {
```

Reply:
Renamed